api.go

```diff
@@ -0,0 +1,137 @@
+package asynqmon
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-redis/redis/v8"
+	"github.com/gorilla/mux"
+
+	"github.com/hibiken/asynq"
+)
+
+// MiddlewareFunc helps chain http.Handler(s).
+type MiddlewareFunc func(http.Handler) http.Handler
+
+type APIOptions struct {
+	RedisConnOpt         asynq.RedisConnOpt
+	Middlewares          []MiddlewareFunc
+	PayloadFormatter     PayloadFormatter
+	StaticContentHandler http.Handler
```

Comment:
Do we need to expose this option? Do users ever need to customize this?

Reply:
I'd like to have the ability to pass staticContents and still be able to use the API as is. If there is some other way you have in mind, I can give feedback. But I don't see any issues with allowing users to modify static contents.

Reply:
I see. But does that mean, we have to document and version the REST API consumed by the frontend? I prefer to keep that internal as I go though rapid iterations (i.e. making frequent breaking changes).

Reply:
I still think this is YAGNI and we should not expose this via public API. 
If we get a feature request from users, I'd revisit this at that point. 
But please drop this option for now.